hnn_core/gui/gui.py

```diff
@@ -98,25 +104,31 @@ def __init__(self, theme_color="#8A2BE2", log_window_height="100px",
                  visualization_window_width="1000px",
                  visualization_window_height="500px",
                  left_sidebar_width='380px',
-                 drive_widget_width="200px"):
+                 drive_widget_width="200px",
+                 header_height="50px",
+                 button_height="30px",
```

Comment:
I think we shouldn't expose all these parameters ... the GUI should be responsive. Everything should be defined as percentages of the total width/height available. Maybe you can allow users to specify the vertical and horizontal split location ... but nothing else

Reply:
I don't think that would be a good idea to make everything responsive ... Responsiveness is good and fancy indeed for web but it also causes problems as some parts in our GUI need a minimal size and the `Layout` in ipywidget is not so flexible that we can customize all CSS attribute. If users think the GUI is too large in the browser page, they should resize the browser.

Reply:
okay let's keep it like this for now ... at least having the layout in one place makes it easy to modify :)